server/games/game.py

```diff
@@ -47,6 +47,10 @@ class GameError(Exception):
     pass
 
 
+class GameClosedError(Exception):
```

Comment:
Probably define a constructor that takes 1 arg. That will be less confusing than using the default constructor plus you can make use of type annotations that way